app/src/main/java/com/example/parkingsystem/mvp/model/ParkingModel.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.parkingsystem.mvp.model
+
+import com.example.parkingsystem.mvp.contract.ParkingContract
+import com.example.parkingsystem.util.Constants
+
+class ParkingModel : ParkingContract.Model {
+    private val parkingSize = Constants.parkingSize
+    override fun getParkingSize(): Int = parkingSize
+}
```

Comment:
to avoid this error add an empty line at the end of the file

Reply:
Done 👍